# Shared patterns of glial transcriptional dysregulation link Huntington's disease and schizophrenia

## **Abstract**
#### Huntingtonâ€™s disease (HD) and juvenile-onset schizophrenia (SCZ) have long been regarded as distinct disorders. However, both manifest cell-intrinsic abnormalities in glial differentiation, with resultant astrocytic dysfunction and hypomyelination. To assess whether a common mechanism might underlie the similar glial pathology of these otherwise disparate conditions, we utilized comparative correlation network approaches to analyze RNA-seq data from human glial progenitor cells (hGPCs) produced from disease-derived pluripotent stem cells. We identified gene sets preserved between HD and SCZ hGPCs yet distinct from normal controls, that included 174 highly-connected genes in the shared disease-associated network, focused on genes involved in synaptic signaling. These synaptic genes were largely suppressed in both SCZ and HD hGPCs, and gene regulatory network analysis identified a core set of upstream regulators of this network, of which OLIG2 and TCF7L2 were prominent. Among their downstream targets, ADGRL3, a modulator of glutamatergic synapses, was notably suppressed in both SCZ and HD hGPCs. ChIP-seq confirmed that OLIG2 and TCF7L2 each bound to the regulatory region of ADGRL3, whose expression was then rescued by lentiviral overexpression of these transcription factors. These data suggest that the disease-associated suppression of OLIG2 and TCF7L2-dependent transcription of glutamate signaling regulators may impair glial receptivity to neuronal glutamate. The consequent loss of activity-dependent mobilization of hGPCs may yield deficient oligodendrocyte production, and hence the hypomyelination noted in these disorders, as well as the disrupted astrocytic differentiation and attendant synaptic dysfunction associated with each. Together, these data highlight the importance of convergent glial molecular pathology in both the pathogenesis and phenotypic similarities of two otherwise unrelated disorders, HD and SCZ. 

## **Scripts** <br>
Analyses in this manuscript were performed following these steps: <br>
[Step 1](https://rawcdn.githack.com/HuynhNPT/shared_dysregulation_in_HD_and_SCZ/private/Script1_Processing-of-individual-networks.html) : Processing of individual networks <br>
[Step 2](https://rawcdn.githack.com/HuynhNPT/shared_dysregulation_in_HD_and_SCZ/private/Script2_Network-preservation.html) : Network preservation <br>
[Step 3](https://rawcdn.githack.com/HuynhNPT/shared_dysregulation_in_HD_and_SCZ/private/Script3_Module-prioritization-for-individual-networks.html) : Module prioritization of individual networks <br>
[Step 4](https://rawcdn.githack.com/HuynhNPT/shared_dysregulation_in_HD_and_SCZ/private/Script4_Identification-of-prioritized-cluster.html) : Identification of prioritized cluster <br>
[Step 5](https://rawcdn.githack.com/HuynhNPT/shared_dysregulation_in_HD_and_SCZ/private/Script5_Downstream-targets-of-OLIG2-and-TCF7L2.html) : Downstream targets of OLIG2 and TCF7L2

## **Raw data accession** <br>
Bulk RNA sequencing data: GSE105041, GSE86906, GSE188558 <br>
ChIP sequencing data: GSE188559, GSE65119, GSE31477
